core/server/models/user.js

```diff
@@ -201,18 +201,13 @@ User = ghostBookshelf.Model.extend({
     // This is used to bypass validation during the credential check, and must never be done with user-provided data
     // Should be removed when #3691 is done
```

Comment:
Is this comment still relevant? 😁

Reply:
Hmm i think so. It's talking about passing `options.validate=false`.
🤔 Not sure where it's used though - will double check.

Reply:
This was originally added when login in, but the e.g. website field was invalid, because we added proper validation after adding the field.

We can remove the option to bypass validation. The validation layer should only validate fields which change.
And if we add a new model, it should check all fields. Commit is coming.